blob/drivertest/drivertest.go

```diff
@@ -224,22 +224,22 @@ func testAttributes(t *testing.T, newHarness HarnessMaker) {
 		}
 	}
 
-	t.Run("ContentType", func(t *testing.T) {
-		b, done := init(t)
-		defer done()
+	for _, rLen := range []int64{0, -1} {
+		t.Run(fmt.Sprintf("ReadLength%d", rLen), func(t *testing.T) {
```

Comment:
nit: keep the name "ReadLength=%d" instead of "ReadLength%d".

Reply:
I will use space.